Book-A-Desk.Api.Tests/BookingsTests.fs

```diff
@@ -32,25 +35,27 @@ let offices =
 let mockGetOffices () =
     offices
     
-let domainName = "domain.com"
+let domainName = "broadsign.com"
     
 let mockReservationCommandFactory : ReservationCommandsFactory =
     {
         CreateBookADeskCommand = fun () -> BookADeskReservationCommand.provide offices domainName
     }
     
 let mutable emailWasSent = false 
-let mockEmailNotification booking =
+let mockEmailNotification _ =
     emailWasSent <- true
     async { return emailWasSent }  
 
 let booking  =
     {
         Office = { Id = mockOfficeId.ToString() }
         Date = DateTime.MaxValue
-        User = { Email = "someEmail" }
+        User = { Email = "someEmail@broadsign.com" }
```

Comment:
Same here, we should avoid putting the name of the company

Reply:
Fixed!